helpers/handlebars/middleware.js

```diff
@@ -33,6 +33,7 @@ const makeActive = (items, currentUrl) => {
 
 module.exports = (req, res, next) => {
     res.locals.themeTitle = process.env.SC_NAV_TITLE || 'Schul-Cloud';
+    res.locals.backendUrl = process.env.BACKEND_URL || 'http://localhost:3030';
```

Comment:
Socket.io won't work if you test on http://127.0.0.1 instead of localhost but that might be unusual